google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -67,12 +71,25 @@
   private JLabel modulesLabel;
   private ModulesComboBox modulesComboBox;
 
+  private final Map<CloudLibrary, CloudApiManagementSpec> apiManagementMap;
   private final List<CloudLibrary> libraries;
+
   private final Project project;
 
+  private static final boolean SHOULD_ENABLE_API_DEFAULT = true;
+
   GoogleCloudApiSelectorPanel(List<CloudLibrary> libraries, Project project) {
     this.libraries = libraries;
     this.project = project;
+
+    apiManagementMap =
+        libraries
+            .stream()
+            .collect(
+                Collectors.toMap(
+                    Function.identity(),
+                    lib -> new CloudApiManagementSpec(SHOULD_ENABLE_API_DEFAULT)));
```

Comment:
Would you mind elaborating a bit on when we might need not to enable an API by default? As I understand, this is for cloud side enable?

Reply:
I think that 'enabled' is a sensible default. But I guess its conceivable that we could determine (after seeing user patterns) that it makes more sense to have the user opt-into having the API enabled (e.g. due to billing ramifications etc.).

Was your point just about exposing that boolean as a member variable?

Reply:
Ah I see. I was just wondering when we actually would want to make the change in that constant, it makes sense.